boltdesignsystem/bolt
# DS-590 Switch button componentv1

## Jira

https://pegadigitalit.atlassian.net/browse/DS-590

## Summary

Adds a new component, Switch Button, which handles boolean UI functions that happens instantly.

## Details

1. Added new component, Jest test and docs.
2. Only bare minimum props are created to handle basic needs for the notifications project. We can expand on this component down the road.

## How to test

Run the branch locally and proof read the Switch Button docs. Make sure the docs are intuitive. Inspect the demo and change on/off to some other language that might be longer, make sure it looks ok. Then check the component code for any mistakes in the code.
